babel.config.js

```diff
@@ -33,9 +33,9 @@ module.exports = {
     plugins: [
         '@babel/plugin-syntax-dynamic-import',
         '@babel/plugin-transform-flow-strip-types',
+        '@babel/plugin-transform-class-properties', // Order matters: `transform-class-properties` must come after `transform-flow-strip-types`
```

Comment:
What happens if this is not in order?

Reply:
was throwing a random error in one of the components. how i understand the issue is that babel was transpiling code in an order which conflicted with code that hadn't been transpiled